src/ctia/properties.clj

```diff
@@ -231,10 +210,19 @@
 (def configurable-properties
   (mls/keys PropertiesSchema))
 
-(def init! (mp/build-init-fn files
-                             PropertiesSchema
-                             ;; TOP-LEVEL STATE!!
-                             (global-properties-atom)))
+(defn build-init-config
+  "Returns a (nested keyword) config map from (merged left-to-right):
+
+  1. [[files]] merged left-to-right
```

Comment:
While `[[foo]]` is also recognised by certain doc tools and IDEs, `#'foo` is less ambiguous since it makes clear that it's a var (and not an arg, or other code reference).

That decreased ambiguity enables stronger tooling (e.g. a refactoring or grep tool) and also eases repl interactions (one can send #'foo to the repl and e.g. `(-> *1 meta :doc)`)